src/test/java/com/basho/riak/client/core/operations/itest/ISearchTestBase.java

```diff
@@ -92,7 +92,7 @@ private static void setupYokoIndexAndBucketProps(RiakClient client, Namespace na
             throws ExecutionException, InterruptedException
     {
         YokozunaIndex index = new YokozunaIndex(indexName);
-        StoreIndex ssi = new StoreIndex.Builder(index).withTimeout(45).build();
+        StoreIndex ssi = new StoreIndex.Builder(index).withTimeout(45000).build();
```

Comment:
Wy don't us make this timeout to be customizable, and use this value as a default, not as a  Magic Number but as a Symbolic Constant like DEFAULT_STORE_INDEX_TIMEOUT